README.adoc

```diff
@@ -51,7 +51,7 @@ sudo apt update
 sudo apt install rdiff-backup
 ----
 
-=== CentOS and RHEL 7 (from EPEL)
+=== CentOS, EuroLinux and RHEL 7 (from EPEL)
```

Comment:
I would simply rephrase that to "RedHat based distribution". No need to list all the RedHat distribution that exists.

And do something similar for Debian, Ubuntu, etc.: "Debian based distribution"

Reply:
I like the term "RHEL Replica" used by @aronowski , it differentiates Centos & Co from Fedora, but else I agree on the principle.

Side note: Red Hat, with a space in-between.

Reply:
Good for me.

Reply:
@aronowski do you mind updating your PR accordingly? You could link to https://en.wikipedia.org/wiki/Red_Hat_Enterprise_Linux_derivatives (I'm also fine with "RHEL derivatives" BTW, to be consistent).

Reply:
@ericzolf Done. Thank you!